Command/IndexRebuildCommand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Massive\Bundle\SearchBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Massive\Bundle\SearchBundle\Search\Event\IndexRebuildEvent;
+use Massive\Bundle\SearchBundle\Search\SearchEvents;
+
+/**
+ * Comand to build (or rebuild) the search index
+ *
+ * @author Daniel Leech <daniel@dantleech.com>
```

Comment:
Actually we agreed on omitting the author tag in all kind of documentation, since it will be wrong at some point anyway.